submissions/lawalNurudeen/html-forms/contact-form.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact-Form</title>
+</head>
+<body>
+    <form action="#">
```

Comment:
@Nurmalee Remove the attribute `action`

Reply:
attribute removed

Reply:
@ToluAkin @TundeOladejo I have done all required/requested changes related this pull request.. kindly process merging concerns so I can proceed to next task/topic ... Thanks